controller/lib/include/stream_port_output_descriptor.h

```diff
@@ -44,5 +45,56 @@ namespace avdecc_lib
          * \return the stream port output descriptor response class.
          */
         AVDECC_CONTROLLER_LIB32_API virtual stream_port_output_descriptor_response * STDCALL get_stream_port_output_response() = 0;
+
+        /**
+         * \return the audio_map get_audio_map response class.
+         */
+        AVDECC_CONTROLLER_LIB32_API virtual stream_port_output_get_audio_map_response * STDCALL get_stream_port_output_audio_map_response() = 0;
+        
+        /**
+         * \param map The added audio_map pending for adding/removal.
+         */
+        AVDECC_CONTROLLER_LIB32_API virtual int store_pending_map(struct audio_map_mapping &map) = 0;
+        
+        /**
+         *  View pending audio mappings.
+         *
+         * \param index The index of the queued audio mapping.
+         */
+        AVDECC_CONTROLLER_LIB32_API virtual int view_pending_maps(size_t index, struct audio_map_mapping &map) = 0;
```

Comment:
Is "view" the correct work here? I think of view == display, but this library does not display anything.